sourcemash/frontend/static/js/views/browse_view.js

```diff
@@ -34,22 +34,28 @@ Sourcemash.Views.BrowseView = Backbone.View.extend({
 
   render: function() {
     // Render parent view
-    this.$el.html(this.template({ models: this.collection.models }));
+    this.$el.html(this.template({ models: this.feedTopicViews }));
```

Comment:
Won't this be empty on the first render? Maybe we should just pass through the topics array and rename models to topics.